### DISCUSSION TITLE
ScriptReference/Physics2D.CapsuleCast

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#632] [Notes]
# ScriptReference/Physics2D.CapsuleCast

https://docs.unity3d.com/ScriptReference/Physics2D.CapsuleCast.html

**Comment by qwe321qwe321qwe321** [comment]
[Physics2D.CapsuleCast works incorrectly with the specific angle of polygon collider.](https://forum.unity.com/threads/physics2d-capsulecast-works-incorrectly-with-the-specific-angle-of-polygon-collider.1509200/) (Fixed in 2022.3.14f1)
